### PR TITLE
Add english tina-cms page (#2681)

### LINKED ISSUE #2681: Add Tina CMS page
As requested in  #2678

### PR DESCRIPTION
**Pull request by feraleyebrows**
#### What kind of changes does this PR include?
- New or updated content

#### Description

- Closes #2681
- What does this PR change? Adds info for Tina CMS
- Did you change something visual? Not really. Just adds content.

### CODE REVIEW

**Comment by sarah11918** [comment]
Hi @feraleyebrows, thanks so much for this contribution!  🥳  You may have noticed that some checks are not passing here. That's because in order to add a new CMS guide, we do need to add a bit of infrastructure around the guide, and not just the content page itself.

Here's a link to an example of a PR that adds a brand new CMS guide so you can see the files that need updating:  https://github.com/withastro/docs/pull/2290/files  Would you be up for making the necessary additions as modeled here? If not, no worries, and one of us in Docs can do it. But, if you can, that would be amazing!  🙌 

Let me know if you want to add the other content, or if you'd rather we take care if it. Thanks!

**Comment by feraleyebrows** [comment]
Hi Sarah,

I've uploaded the svg and tweaked the cms. Whilst I was there, I used the sort lines function in sublime to sort all the lines in the logos.ts file into alphabetical order. I think this makes more sense but I understand if you don't want it done or would prefer it in in a separate commit because of potential for merge conflicts so let me know if you want me to revert it back.

**Comment by sarah11918** [comment]
Thanks @feraleyebrows! I am going to say please revert the alpha sorting, thanks. I'm not the one who established the order of that list, and I have a feeling that if they had wanted it in alpha order, they would have done so since they are usually pretty intentional about their choices.  :sweat_smile: 

It may be a good idea, and I'll mention it for the future, but let's just try to fit things in seamlessly for now, in this PR! I will bring it up, though!

**Comment by feraleyebrows** [comment]
no worries. done.

**Comment by Jutanium** [comment]
Hey @feraleyebrows,
Thanks so much for this PR! Are you happy with us taking over the PR, editing for clarity and making sure it fits with our other guides? We'll get this merged in no time.

**Comment by feraleyebrows** [comment]
yeah no worries

**Comment by Jutanium** [comment]
@sarah11918 - tested, revised, and ready for your review!

**Review by sarah11918 — COMMENTED** [review]
Looking good, everyone! Thanks for all the work that has gone into this! Just a few suggestions from me in final edit stage, but looking forward to seeing this merged!  🙌

File: src/content/docs/en/guides/cms/tina-cms.mdx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      // package.json
+      {
+          "scripts": {
+              "dev": "astro dev",
+              "dev": "yarn tinacms dev -c 'astro dev'"
+          }
+      }
+      ```
+      </Fragment>
+    </PackageManagerTabs>
```
Comment (Jutanium) [inline src/content/docs/en/guides/cms/tina-cms.mdx:78]:
```suggestion
    <PackageManagerTabs>
      <Fragment slot="npm">
      ```json del={4} ins={5}
      // package.json
      {
          "scripts": {
              "dev": "astro dev",
              "dev": "npm tinacms dev -c 'astro dev'"
          }
      }
      ```
      </Fragment>
      <Fragment slot="pnpm">
      ```json del={4} ins={5}
      // package.json
      {
          "scripts": {
              "dev": "astro dev",
              "dev": "pnpm tinacms dev -c 'astro dev'"
          }
      }
      ```
      </Fragment>
      <Fragment slot="yarn">
      ```json del={4} ins={5}
      // package.json
      {
          "scripts": {
              "dev": "astro dev",
              "dev": "yarn tinacms dev -c 'astro dev'"
          }
      }
      ```
      </Fragment>
    </PackageManagerTabs>
```

**Pull request merged**